### DISCUSSION TITLE
Controllo Luci con relè a più uscite

### DISCUSSION THREAD

**Discussion by NTorto** [indomus/forum#340] [Hardware & Firmware]
Ciao a tutti,
sto iniziando ad avvicinarmi al mondo della domotica e di Home Assistant.

Vorrei controllare le luci della sala: due gruppi di luci sono collegate ad un relè a due uscite che tramite pulsanti si attivano alternativamente. Per capirci come nell'immagine allegata: ad una pressione (di qualunque pulsante in sala) si accendono entrambe, alla seconda pressione si spegne un gruppo ecc.

Io vorrei mantenere la logica di accensione ma domotizzarla, lasciando comunque la possibilità di utilizzare i pulsanti esistenti, e volendo avere il feedback di quale dei due gruppi è acceso. 

Ho dato un'occhiata a qualche shelly o sonoff ma non ho trovato quello adatto al mio caso. La cosa più vicina che ho trovato è utilizzare un sonoff con la funzione di inching per dare l'impulso al relè ma non avrei il feedback di quale gruppo di luci è acceso o nemmeno se sono tutte spente.

Vi ringrazio per come saprete consigliarmi.

![immagine](https://user-images.githubusercontent.com/5862639/119009432-66be2300-b993-11eb-8afc-fa1bbdfaefc9.png)

**Comment by Larry-78** [comment]
Ciao, visto che si tratta di un relè ad impulsi (detto anche commutato) a 2 contatti, io ho risolto utilizzando 2 sonoff mini. In pratica basterebbe sostituire il relè con uno di tipo Finder 26 dove i due contratti puliti andranno collegati ai relativi morsetti S1 e S2 dei mini insieme alle fasi che alimentano i due gruppi lampade. In questo modo il pulsante farà eccitare la bobina grazie al collegamento standard del relè e i contatti faranno cambiare stato ai mini permettendoti il controllo con HA. Io però mi sono trovato in fase di ristrutturazione e quindi ho provveduto a installare cassette più grandi perché ovviamente un relè e 2 mini hanno il loro ingombro, ora non so in quale situazione ti trovi.

**Reply by NTorto** [reply]
Perfetto, ottima soluzione, ti ringrazio molto. Ho verificato ed ho già il finder 26.04 nella scatola di derivazione che è abbastanza spaziosa, credo di riuscire a piazzarci tranquillamente i due sonoff. Ti ringrazio molto per l'aiuto.

**Reply by Larry-78** [reply]
Di nulla è un piacere.
Ti ho consigliato di sostituirlo perché nella maggior parte dei casi si utilizza un Finder 27 che è più compatto e il morsetto della fase è in comune ai contatti.
